perception/autoware_tensorrt_bevdet/include/autoware/tensorrt_bevdet/data.hpp

```diff
@@ -0,0 +1,72 @@
+// Copyright 2024 AutoCore, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// cspell:ignore BEVDET, intrin, Quater, BEVDET
+#ifndef AUTOWARE__TENSORRT_BEVDET__DATA_HPP_
```

Comment:
Consider replacing the header guards `#ifndef AUTOWARE__TENSORRT_BEVDET__DATA_HPP_` and `#define AUTOWARE__TENSORRT_BEVDET__DATA_HPP_` with a more concise and less error-prone form, such as `#pragma once`.  This approach is generally preferred for its simplicity and reduced risk of naming collisions.